src/SourceBuild/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/SdkContentTests.cs

```diff
@@ -46,10 +47,33 @@ private void WriteTarballFileList(string? tarballPath, string outputFileName, bo
         fileListing = BaselineHelper.RemoveRids(fileListing, isPortable);
         fileListing = BaselineHelper.RemoveVersions(fileListing);
         IEnumerable<string> files = fileListing.Split(Environment.NewLine).OrderBy(path => path);
+        files = RemoveExclusions(
+                    files,
+                    GetExclusionFilters(
+                        Path.Combine(BaselineHelper.GetAssetsDirectory(), "SdkDiffExclusions.txt"),
+                        sdkType));
 
         File.WriteAllLines(outputFileName, files);
     }
 
+        private static IEnumerable<string> RemoveExclusions(IEnumerable<string> files, IEnumerable<string> exclusions) =>
+            files.Where(item => !exclusions.Any(p => FileSystemName.MatchesSimpleExpression(p, item)));
+
+        private static IEnumerable<string> GetExclusionFilters(string exceptionFilePath, string sdkType)
+        {
+            string[] lines = File.ReadAllLines(exceptionFilePath);
```

Comment:
For brevity, consider in-lining below

```
return File.ReadAllLines(exceptionFilePath)
                .Where(line => line.StartsWith(sdkType + ",")) // process only specific sdk exclusions
                .Select(line =>
                {
                    // Ignore comments
                    var index = line.IndexOf('#');
                    return index >= 0 ? line[prefixSkip..index].TrimEnd() : line[prefixSkip..];
                })
                .ToList();
```

Reply:
Fixed with https://github.com/dotnet/installer/pull/15650/commits/0048bcd04618fd37255c405266a7c9b64cafce07